.github/workflows/ci.yml

```diff
@@ -13,8 +13,6 @@ jobs:
     strategy:
       matrix:
         cfg:
-          - {puppet_version: '7', ruby: '2.7'}
-          - {puppet_version: '7', ruby: 'jruby-9.3.7.0'}
```

Comment:
Two potential issues are:

1. We need to maintain compatibility with older agents that haven't upgraded yet. We always want to support upgrading old agents
1. s puppet types and providers may be loaded in puppetserver and JRuby 9.4 in puppet 8 maintains compatibility with MRI 3.1 language features. So resource-api can't use any ruby language features that were added in MRI 3.2. As long as we continue to test with rubocop nightly then it should be ok to drop 

As long as we run rubocop as part of our CI check that that should be ok to remove puppet7 completely